app/views/applicant_mailer/unassignment.html.haml

```diff
@@ -0,0 +1,11 @@
+%p
+  Dear #{full_name(@applicant)},
+
+%p
+  You have been unassigned as an applicant on the submission
```

Comment:
"removed" is probably a better term here. Also, as above, I think the name of the person doing the removing should be included.